.tool-versions.sample

```diff
@@ -0,0 +1,2 @@
+ruby 2.6.6
```

Comment:
Should we start with v2.5 as it's the one we have the tests running on or better push dev to 2.6 already and thus force the upgrade once things start breaking in CI?

Reply:
I'd try to leave old versions and push as much to the edge ;)
As long as it is working it is a good way also to push for 2.6 in production